ref-cache.md

```diff
@@ -2,3 +2,50 @@
 permalink: /ref-cache.html
 ---
 
+# Loc-I Reference Integrated Cache
+
+## What
+
+Central reference cache that loads a set of reference spatial layers directed by the Loc-I steering committee.
+     
+
+
+## Why
+
+The Loc-I integrated graph cache provides a graph store for applications and users to query spatial linked data features. This provides users with SPARQL APIs that allows for flexible and powerful queries across the graph store (triple store).
```

Comment:
'Loc-I reference cache' try to use the same phrase as a name, consistently

Reply:
done